server/mscalendar/welcome_flow.go

```diff
@@ -6,15 +6,15 @@ import (
 	"github.com/mattermost/mattermost-plugin-mscalendar/server/utils/flow"
 )
 
-type welcomeFlow struct {
+type WelcomeFlow struct {
```

Comment:
Does this need to become public?

Reply:
I don't know what is this, in vs code it shows no change 
![image](https://user-images.githubusercontent.com/7377334/99186001-fb7e6480-2755-11eb-8934-a415cb78cd4a.png)

Reply:
Because `NewWelcomeFlow` returns a `WelcomeFlow`. Go lint insists of exporting structs that are returned.